STTemplate-Tests/STTemplateTest.class.st

```diff
@@ -867,13 +930,14 @@ STTemplateTest >> testSourceForEnclosingNoCodeBeforeElement [
 
 	| template |
 	template := '<st 6+5 ><p>Hello</p>' asSTTemplate.
-	self assert: template asSmalltalkSource equals: '| out result |
+	self assert: template asSmalltalkSource equals: '_anonymousRenderer
+| out result |
 out := String new writeStream.
```

Comment:
@fgarau I like `streamContents:` a lot too but can't be used here because it needs to return the stream.
That stream is used elsewhere. For example `STTCurrentRenderingStream` has it for continuing rendering in that stream the Smalltalk source code needed for a sub-template (a "partial").

Thank you for the review!!